### PR TITLE
Add Modify Partition Button That Opens Gparted

### PR DESCRIPTION
**Pull request by mmstick**
When the button is used, gparted will be opened and the UI locked. When Gparted is closed, the disk bars will be destroyed and re-generated, the next button reset to a false sensitivity, and any active mount points reset.

### CODE REVIEW

File: src/Views/PartitioningView.vala [hunk 1]
```diff
@@ -48,6 +50,24 @@ public class Installer.PartitioningView : AbstractInstallerView  {
         this.content_area.attach(description, 0, 0, 1, 1);
         this.content_area.attach(disk_scroller, 0, 1, 1, 1);
 
+        load_disks ();
+
+        gparted_button = new Gtk.Button.with_label (_("Modify Partitions"));
+        gparted_button.get_style_context ().add_class (Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
```
Comment (jackpot51) [inline src/Views/PartitioningView.vala]: I do not know that having two destructive action buttons is good

Comment (jackpot51) [reply src/Views/PartitioningView.vala]: Yes

Comment (mmstick) [reply src/Views/PartitioningView.vala]: It is done

**Comment by jackpot51** [comment]
I had to revert this, we need to rebase it once the new layout for the partitioning page is done.

**Pull request merged**